java/athena/src/main/java/sleeper/athena/record/SleeperRecordHandler.java

```diff
@@ -124,22 +124,22 @@ protected void readWithConstraint(BlockSpiller spiller, ReadRecordsRequest recor
      * Implementation dependent code to create the schema used to read the data. Some implementations may be able to
      * slim down the schema to reduce the amount of data read per query, thereby making queries cheaper.
      *
-     * @param schema         the original schema associated with the table being queried
-     * @param recordsRequest the records request made by the user
-     * @return a schema to use for reading the files.
+     * @param  schema         the original schema associated with the table being queried
+     * @param  recordsRequest the records request made by the user
+     * @return                a schema to use for reading the files.
      */
     protected abstract Schema createSchemaForDataRead(Schema schema, ReadRecordsRequest recordsRequest);
 
     /**
      * Implementation dependent iterator creation code. The entire request which contains the user, split and schema is
      * passed to this method along with the table properties.
      *
-     * @param recordsRequest  the request
-     * @param schema          the table schema to use for reading
-     * @param tableProperties The table properties to use for reading the table
-     * @return an iterator of records
-     * @throws Exception when an iterator is not created
-     * @implNote Do not use the schema in the table properties as it could differ from the schema provided.
+     * @param    recordsRequest  the request
+     * @param    schema          the table schema to use for reading
+     * @param    tableProperties The table properties to use for reading the table
+     * @return                   an iterator of records
+     * @throws   Exception       when an iterator is not created
+     * @implNote                 Do not use the schema in the table properties as it could differ from the schema provided.
```

Comment:
There is a bit of inconsistency with the capitalisation of sentences after javadoc tags. We could either leave them all lowercase and treat the description like a note, or capitalise the start of the note and make them full sentences.

Making the description a full sentence and capitalising the start of it might be better for parameters which only have a space between the parameter name and the description, like the tableProperties parameter above